src/installer/test/TestUtils/HostFxrVersion.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace TestUtils
+{
+    public class HostFxrVersion
```

Comment:
I'm not sure having the separate class is really necessary / how much it really provides over just using the `Version` class.